pycbc/events/stat.py

```diff
@@ -1593,6 +1600,9 @@ def rank_stat_single(self, single_info):
         # Combine the signal and noise rates
         loglr = ln_s - ln_noise_rate
 
+        # Apply statistic correction if given
```

Comment:
```suggestion
        # Apply statistic correction
```
It's always applied (the fact that default is 0 is explained elsewhere)